source/jormungandr/jormungandr/interfaces/v1/serializer/fields.py

```diff
@@ -146,6 +146,7 @@ class LinkSchema(serpy.Serializer):
     href = StrField()
     value = StrField()
     category = StrField()
+    comment_type = StrField()
```

Comment:
I do not want to display it when None.
This object is only used for the schema, not for serializing the object.
I will check that the schema is correct but I think it is since test doesn't fail.